spark2/src/test/java/org/apache/iceberg/examples/SchemaEvolutionTest.java

```diff
@@ -22,18 +22,27 @@
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
 import org.apache.commons.io.FileUtils;
 import org.apache.iceberg.PartitionSpec;
 import org.apache.iceberg.Schema;
 import org.apache.iceberg.Table;
 import org.apache.iceberg.hadoop.HadoopTables;
 import org.apache.iceberg.types.Types;
+import org.apache.spark.sql.Column;
 import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.QueryTest$;
 import org.apache.spark.sql.Row;
 import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.catalyst.expressions.Literal$;
 import org.apache.spark.sql.types.DataTypes;
-import org.junit.Before;
-import org.junit.Test;
+import org.apache.spark.sql.types.LongType$;
+import org.apache.spark.sql.types.StructField;
+import org.junit.*;
```

Comment:
Style: no wildcard imports because it isn't obvious where symbols are coming from.

Reply:
done